syntax_highlighting/fr/catala_fr.iro

```diff
@@ -216,7 +216,7 @@ code : context {
   }
 
   : pattern {
-    regex \= \b(selon|sous\s+forme|fixé|par|décroissante|croissante|varie|avec|on\s+a|soit|dans|champ\s+d'application|dépend\s+de|déclaration|inclusion|contenu|règle|sous\s+condition|condition|donnée|conséquence|rempli|égale\s+à|assertion|définition|état|étiquette|exception)\b
+    regex \= \b(selon|sous\s+forme|fixé|par|décroissante|croissante|varie|avec|on\s+a|soit|dans|champ\s+d'application|dépend\s+de|déclaration|inclusion|contenu|règle|sous\s+condition|condition|donnée|conséquence|rempli|égal\s+à|assertion|définition|état|étiquette|exception)\b
```

Comment:
YES this was broken for me.

Reply:
Could you check whether all new keywords are in there? For instance `résultat` is missing for me.

Reply:
You need to rebuild the pygments lexer (`make pygmentize_fr`), all should be there
(at least all that is in the cheat-sheet, that's how I checked :p )